src/react/account/AccountUserAccessKeys.tsx

```diff
@@ -167,8 +168,8 @@ const AccountUserAccessKeys = () => {
     status: accessKeysStatus,
   } = useAwsPaginatedEntities(
     {
-      queryKey: ['listIAMUserAccessKey', IAMUserName],
-      queryFn: (_ctx, marker) => IAMClient.listAccessKeys(IAMUserName, marker),
+      queryKey: getUserAccessKeysQuery(IAMUserName, IAMClient).queryKey,
```

Comment:
Do you think we should verify the `IAMClient` is not empty?
```suggestion
      queryKey: getUserAccessKeysQuery(IAMUserName, notFalsyTypeGuard(IAMClient)).queryKey,
```

Reply:
I think it's not mandatory because the query is already based on this condition "enabled: IAMClient !== null". 
But, we can add it to double-check if IAMClient is null or not.